### PR TITLE
New search page

### PR DESCRIPTION
**Pull request by talhasch**

### CODE REVIEW

File: src/common/routes.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default {
     EDIT: `/:username(@[\\w\\.\\d-]+)/:permlink/edit`,
     EDIT_DRAFT: `/draft/:draftId`,
     DISCOVER: `/discover`,
+    SEARCH_MAIN: `/search-main/`,
```
Comment (feruzm) [inline src/common/routes.ts:32]: Why not just search?

File: src/common/i18n/locales/en-US.json [hunk 2]
```diff
@@ -670,13 +673,45 @@
...
+    "date": "Date",
+    "date-week": "Week",
+    "date-month": "Month",
+    "date-year": "Year",
+    "date-all": "All",
+    "sort": "Sort",
     "sort-popularity": "Popularity",
     "sort-relevance": "Relevance",
-    "sort-newest": "Date"
+    "sort-newest": "Date",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:703]: This could be `Recent` instead of Date

File: src/common/i18n/locales/en-US.json [hunk 3]
```diff
@@ -670,13 +673,45 @@
...
+    "hide-low": "Hide low quality content",
```
Comment (feruzm) [inline src/common/i18n/locales/en-US.json:704]: `Hide low payout content` better because `quality` is subjective

Comment (talhasch) [reply src/common/i18n/locales/en-US.json]: done

File: src/common/i18n/locales/en-US.json (hunk 2 above)
Comment (talhasch) [reply src/common/i18n/locales/en-US.json]: done

File: src/common/routes.ts (hunk 1 above)
Comment (talhasch) [reply src/common/routes.ts]: changed routing structure.

`/search` is main search page.

`/search-more` allows scrolling into search results.

Comment (feruzm) [reply src/common/routes.ts]: `/search-main` you mean right?

**Review by feruzm — APPROVED** [review]
looking great 🚀

File: src/common/routes.ts (hunk 1 above)
Comment (talhasch) [reply src/common/routes.ts]: removed `/search-main` created `/search-more` instead. sounds like more appropriate :)

**Pull request merged**